hack/import-restrictions.json

```diff
@@ -403,38 +399,6 @@
     "allowedImportPackages": []
   },
 
-  {
-    "checkedPackageRoots": [
-      "github.com/openshift/origin/pkg/templateservicebroker"
-    ],
-    "allowedImportPackageRoots": [
-      "vendor/k8s.io/apimachinery",
-      "vendor/k8s.io/apiserver",
-      "vendor/k8s.io/client-go",
-      "vendor/github.com/spf13/cobra",
-      "vendor/golang.org",
-      "vendor/github.com/openshift/api",
-      "vendor/github.com/openshift/client-go",
-      "github.com/openshift/origin/pkg/template/servicebroker"
-    ],
-    "allowedImportPackages": [
-      "vendor/k8s.io/klog",
-      "vendor/k8s.io/api/core/v1",
-      "vendor/k8s.io/api/authorization/v1",
-      "vendor/k8s.io/kubernetes/pkg/apis/core",
-      "vendor/k8s.io/kubernetes/pkg/apis/core/install",
-      "vendor/github.com/emicklei/go-restful",
-      "vendor/github.com/lestrrat/go-jsschema",
-      "vendor/k8s.io/kube-aggregator/pkg/apiserver",
-      "vendor/k8s.io/kubernetes/pkg/controller",
-      "vendor/k8s.io/kubernetes/pkg/kubectl/cmd/util",
-      "github.com/openshift/origin/pkg/api",
-      "github.com/openshift/origin/pkg/template/apis/template",
-      "github.com/openshift/origin/pkg/template/apis/template/validation",
-      "github.com/openshift/origin/pkg/api/legacy"
-    ]
-  },
-
```

Comment:
can we update them instead of removing?

Reply:
We want to move towards `go list` in the long run, so this will eventually die.